### PR TITLE
Redesigned FAQs page

### PR DESCRIPTION
**Pull request by Ed-Gah**
## Proposed Changes
- Created a faqs page
- Implemented a display component for the FAQs and answers
- Implemented an approach to persist Navbar and footer on the respective positions

## Files Added
- `components/FaqsSection.js`
- `pages/faqs.js`

## File Updated
- `components/Header.js`.

## Checked
Codebase structure followed :+1:

## Visual Results
Screenshot of proposed implementation of Redesigning FAQs page

### CODE REVIEW

**Comment by Ed-Gah** [comment]
### Fixes based on comments

## File Restored
`.env.example`

## Files updated
`components/FaqsSection.js`
`pages/faqs.js`
`package.json`

## Visual Results
Screenshot from 2023-10-05 10-28-29

**Comment by Parthiv-M** [comment]
The `pages/api/auth/[...nextauth].js` does not have to be changed. Those details are only for the dev environment

**Comment by Ed-Gah** [comment]
## Fixes on task [T340385](https://phabricator.wikimedia.org/T340385) based comments

## File Updated
`pages/api/auth/[...nextauth].js

**Comment by Parthiv-M** [comment]
Couple of minor edits, otherwise good to go

**Comment by Parthiv-M** [comment]
Couple of minor edits to the content, rest is good to go
- **_to be archived(which)_** to be changed to **_to be archived (which)_**
- **_So it doesn't support_** to be changed to  **_So it is not supported_** 
- Under _What are the future enhancements?_, the answer can be changed to **_Bulk upload feature, direct upload to Wikimedia Commons and addition of newer libraries are some features being actively worked upon_**

**Comment by Ed-Gah** [comment]
Minor edits done Sir

**Review by Parthiv-M — APPROVED** [review]
LGTM

File: pages/api/auth/[...nextauth].js
Comment (coderwassananmol) [inline pages/api/auth/[...nextauth].js:1]: Please undo the changes made to this file.

**Review by coderwassananmol — CHANGES_REQUESTED** [review]
Please pull the latest changes from the `develop` branch and make the requested change. Rest is good to go. :)

**Comment by Ed-Gah** [comment]
Done Sir

File: pages/api/auth/[...nextauth].js [hunk 1]
```diff
@@ -7,8 +7,8 @@ export const authOptions = {
   providers: [
     WikimediaProvider({
       clientId: process.env.WIKIMEDIA_CLIENT_ID,
-      clientSecret: process.env.WIKIMEDIA_CLIENT_SECRET
-    })
+      clientSecret: process.env.WIKIMEDIA_CLIENT_SECRET,
```
Comment (coderwassananmol) [inline pages/api/auth/[...nextauth].js:10]: @Ed-Gah Please remove "," from here.

File: pages/api/auth/[...nextauth].js [hunk 2]
```diff
@@ -7,8 +7,8 @@ export const authOptions = {
...
+    }),
```
Comment (coderwassananmol) [inline pages/api/auth/[...nextauth].js:11]: Please remove "," from here.

**Review by coderwassananmol — CHANGES_REQUESTED** [review]

**Comment by Ed-Gah** [comment]
Actually Sir, its removed since yesterday. But  when to commit and push husky sees it as an empty commit
Screenshot from 2023-10-09 09-24-12

**Comment by coderwassananmol** [comment]
Okay. Closing this then.

**Pull request closed without merging**